### PR TITLE
[T-52] Users&Roles FE

### PR DESCRIPTION
**Pull request by lakatop**
Functionality added:
* List of users
* List of roles
* Delete functionality for both lists + snackbar + instant list refresh
* Functioning dummy routes for other actions and adding new user/role

https://user-images.githubusercontent.com/61156764/231844123-ba0a0bb8-6462-4937-a90a-72dc4e665007.mov

### CODE REVIEW

File: src/backend/core/roles/views.py
Comment (michalpulpan) [inline src/backend/core/roles/views.py:1]: Yeah, this was mentioned in my previous comment, it's a view `DeleteGroup` implementing `post` method 😆

File: src/backend/core/roles/urls.py
[the remark this answers is not in the document]
Comment (VL-CZ) [reply src/backend/core/roles/urls.py:1]: I agree, we should follow *standard* REST practices
Therefore
POST `/group` -> create new group
DELETE `/group/{id}` -> delete the group
GET `/group/{id}` -> get group detail
...

[the remark this answers is not in the document]
Comment (lakatop) [reply src/backend/core/roles/urls.py:1]: Changed.

File: src/dashboard/pages/dashboard/users-roles/index.tsx [hunk 1]
```diff
@@ -31,8 +36,9 @@ DashboardRolesAndUsersPage.getLayout = (page: ReactElement) => {
   );
 };
 
-export const getServersideProps = async (context: any) => {
-  console.log("Dashboard orders");
+export const getServerSideProps: GetServerSideProps = async (context) => {
```
[the remark this answers is not in the document]
Comment (lakatop) [reply src/dashboard/pages/dashboard/users-roles/index.tsx]: Deleted.

**Pull request merged**